app/build.gradle

```
@@ -86,6 +88,11 @@ dependencies {
 
     testImplementation 'org.robolectric:robolectric:4.3.1'
     testImplementation 'androidx.test:runner:1.3.0'
+
+    implementation "com.google.dagger:hilt-android:2.31-alpha"
+    kapt 'com.google.dagger:hilt-android-compiler:2.31-alpha'
+    implementation 'androidx.hilt:hilt-lifecycle-viewmodel:1.0.0-alpha02'
+    kapt 'androidx.hilt:hilt-compiler:1.0.0-alpha02'
 }
 
```

Kotlin で Hilt を使う場合は `correctErrorTypes` を `true` にしたほうがよさそう。
（よくわかっていないので違ったら指摘お願い）
ref: https://dagger.dev/hilt/gradle-setup

```suggestion

kapt {
    correctErrorTypes = true
}

```

---

最初入れていて、入れなくてもビルドが通ったから消してしまっていた。。。
ガーも正直わからんがドキュメントにあるしいれよう！！